### PR TITLE
Gql01 user create ap itests zhemanova

### PR DESCRIPTION
**Pull request by Nataliia-Zhemanova**
userCreate API tests

### CODE REVIEW

File: src/modules/user/_test/userCreateZhe.spec.js [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+        variables: arg,
+      };
+      request(graphQLEndpoint)
+        .post("/")
+        .send(postData)
+        .expect(200) //supertest
+        .end((err, res) => {
+          if (err) return done(err);
+          const respData = res.body;
+          console.log("RESP BODY ===", respData);
```
Comment (olgatbird) [inline src/modules/user/_test/userCreateZhe.spec.js:31]: Natalia, please, delete console.log

File: src/modules/user/_test/userCreateZhe.spec.js [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+  userCreate(userInput: $userInput) {
+    _id
+    firstName
+    lastName
+  }
+}`,
+        variables: arg,
+      };
+      request(graphQLEndpoint)
+        .post("/")
```
Comment (olgatbird) [inline src/modules/user/_test/userCreateZhe.spec.js:25]: Please, optimize your code

File: src/modules/user/_test/userCreateZhe.spec.js [hunk 3]
```diff
@@ -0,0 +1,39 @@
...
+        .send(postData)
+        .expect(200) //supertest
+        .end((err, res) => {
+          if (err) return done(err);
+          const respData = res.body;
+          console.log("RESP BODY ===", respData);
+          // expect(respData.userCreate.firstName).eq("firstName1");
```
Comment (olgatbird) [inline src/modules/user/_test/userCreateZhe.spec.js:32]: Please, add expects and delete comments

File: src/modules/user/_test/userCreateZhe.spec.js [hunk 4]
```diff
@@ -0,0 +1,39 @@
...
+          // expect(respData.userCreate.lastName).eq("lastName1");
+          done();
+        });
+    });
+  });
+  describe("USER CREATE - NEGATIVE", () => {});
```
Comment (olgatbird) [inline src/modules/user/_test/userCreateZhe.spec.js:38]: Please, create negative tests

**Review by olgatbird — CHANGES_REQUESTED** [review]

File: src/modules/user/_test/userCreateZhe.spec.js (hunk 3 above)
Comment (Nataliia-Zhemanova) [reply src/modules/user/_test/userCreateZhe.spec.js:32]: Done!

File: src/modules/user/_test/userCreateZhe.spec.js (hunk 4 above)
Comment (Nataliia-Zhemanova) [reply src/modules/user/_test/userCreateZhe.spec.js:38]: Done!

**Pull request closed without merging**